MediaBrowser.Model/Session/GeneralCommandType.cs

```diff
@@ -48,6 +48,7 @@ public enum GeneralCommandType
         PlayNext = 38,
         ToggleOsdMenu = 39,
         Play = 40,
-        SetMaxStreamingBitrate = 41
+        SetMaxStreamingBitrate = 41,
+        SetOrderMode = 42
```

Comment:
I think we can re-use `SetShuffleQueue` for now.

Reply:
I think the client code would get really confusing without the new command if we only add PlaybackOrder as a field: 
<img width="365" alt="image" src="https://github.com/jellyfin/jellyfin/assets/21142851/5b2b8b98-f6a4-4d88-9526-a29bd56a26c7">
It would look like the setter is changing one field, but getter is returning another.

By adding the new command, we could start gradually replacing `SetShuffleQueue` in client code. When would be better time to change it than now, when we will be changing getters for shuffleMode in clients?

Reply:
Fair enough, let's add it!